Configuration/Generator/python/XiMinus_13p6TeV_SoftQCDInel_TuneCP5_cfi.py

```diff
@@ -4,7 +4,7 @@
 from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
```

Comment:
Is this for Run III or UL? If it is the former, this line needs to be changed.

Reply:
`Configuration.Generator.MCTunesRun3ECM13p6TeV.PythiaCP5Settings_cfi import *`

Reply:
Yes, if that's the prescription from generators we are good with that. I have no clue if this may have some side effects, such as changing filter efficiencies.